### ISSUE TITLE
[Bug]: Unable to install the bootstrap packages and causes TouchWiz Home to crash

### ISSUE THREAD

**Issue by umuto7281** [termux/termux-app#2966]
### Problem description

I tried installing termux on my Samsung Galaxy J2 (SM-J200G)
Although there was internet connection, i got an error saying that Termux was unable to install the bootstrap packages.
And right after that TouchWiz Home kept crashing repeatedly until i uninstalled Termux.
Tried to use a version that is older than v0.83 (tried v0.73),but still no luck.
Yeah, that's all i can say.
(Forgot to mention, the device is rooted. Idk if that changes anything)

### Steps to reproduce the behavior.

Basically downloading/installing Termux.

### What is the expected behavior?

Termux not being able to install its bootstrap packages, and TouchWiz Home repeatedly crashing.

### System information

* Termux application version:0.83,also tried version 0.73
* Android OS version:5.1.1
* Device model: Samsung SM-J200G

**Comment by agnostic-apollo** [comment]
https://github.com/termux/termux-app/wiki/Termux-on-android-5-or-6
